src/main/java/cuchaz/enigma/Main.java

```diff
@@ -95,8 +79,8 @@ public static void main(String[] args) throws IOException {
 		}
 	}
 
-	private static class PathConverter implements ValueConverter<Path> {
-		static final ValueConverter<Path> INSTANCE = new PathConverter();
+	public static class PathConverter implements ValueConverter<Path> {
```

Comment:
Use an enum for singleton?

Reply:
using enums for singletons is to me an abuse of what enums are actually meant to be used for

Reply:
I personally like the pattern of using enums for singletons, but it seems to be a divisive one, and not necessary for this change set (it's only someone else's singleton I'm making public).